### ISSUE TITLE
`Storefront_Custom_Radio_Image_Control` customizer buttons don't work in IE11

### ISSUE THREAD

**Issue by jconroy** [woocommerce/storefront#1476]
## Describe the bug
Customizer controls that are built from `Storefront_Custom_Radio_Image_Control` don't appear to work in IE11.

Examples of controls that implement this include Storefront General Layout and Storefront Reviews layout

![Screen Shot 2020-09-21 at 2 41 27 pm](https://user-images.githubusercontent.com/355014/93734519-aff97f80-fc18-11ea-87b3-f8d4c19079d6.png)

![Screen Shot 2020-09-21 at 2 41 47 pm](https://user-images.githubusercontent.com/355014/93734524-b4259d00-fc18-11ea-948b-77ddeaac483a.png)

Isolating the problem (mark completed items with an [x]):
- [x] I have deactivated other plugins and themes and confirmed this bug occurs when only WooCommerce + Storefront theme are active.
- [x] I can reproduce this bug consistently using the steps below.

## To Reproduce
1. Use IE11
2. Open customizer
3. Got to "Layout"
4. Attempt to select an alternative option - clicking the layout doesn't do anything - no error logs are recorded.

## Expected behavior
Selecting a layout option previews within the site customizer and the option is selected.

### Browser Environment
Please provide as much detail as possible about your testing environment.

- Platform: e.g. Windows 10 Pro
- Browser(s): IE 11 - 11.508.xxx

**Comment by haszari** [comment]
Dang! Perhaps we have support a more limited range of browsers for admin – is this control used on the front end anywhere? But that's a bit of a cop-out, especially since this is silently failing in IE11. 

I don't see anything in the [history for this file](https://github.com/woocommerce/storefront/commits/a5f9c57962be2628b4f1d3c2ed5318612897fb58/inc/customizer/class-storefront-customizer-control-radio-image.php) suggesting the lack of IE support is intentional, so labelling high priority. 

Bottom line is that users in supported browsers should be able to use these settings (sidebar/reviews layout).

**Comment by opr** [comment]
Based on [this comment](https://make.wordpress.org/core/2021/03/04/discussion-dropping-support-for-ie11/#comment-40870) by Matt in favour of dropping IE11 support in WP Core, is this something we still want to work on?  I don't think it is so I will close the issue. Please feel free to continue the discussion if you have any further thoughts though!
